### ISSUE TITLE
can't find pyinstaller.exe when run build_windows_executable.bat

### ISSUE THREAD

**Issue by zxt5105515** [MestreLion/git-tools#79]
env:
windows 11
python 3.12.3
pyinstaller 6.8.0

after install i can't find any file named pyinstaller.exe with everything, then found that i need change all ```pyinstaller.exe``` to ```python -m PyInstaller``` in build_windows_executable.bat

**Comment by MestreLion** [comment]
Yeah, this is mentioned in PyInstaller's docs, it has its own section about this, [Troubleshooting missing PyInstaller command](https://pyinstaller.org/en/latest/installation.html#troubleshooting-missing-pyinstaller-command):

> If you cannot use the `pyinstaller` command due to the scripts directory not being in `PATH`, you can instead invoke the `PyInstaller` module, by running `python -m PyInstaller`

I should do this change, either add this to the documentation, or better yet in `build_windows_executable.bat` itself
